### ISSUE TITLE
GHG Landing Page

### ISSUE THREAD

**Issue by aboydnw** [NASA-IMPACT/veda-config-ghg#48]
## Description

The GHG team received some direction from HQ on the landing page. They specifically pointed to [this page](https://eotoolkit.unhabitat.org/) as inspiration. some specific notes on direction they gave us:
1. They want to choose a visually interested picture for the banner. They have some ideas on which image to choose, and will provide it to us. Ideally, this would be a gif (is that possible in the banner?)
2. They want to have some "buckets" at the top of the page that outline the 3 major areas within GHG. These would directly correlate to data stories or other content in GHG
3. They like the idea of showing some sort of feed of recent or notable content
4. Another idea from Manil was to show some analytics or metrics of some kind.

We walked through the current VEDA landing page to show the three bubbles we have at the top and the Featured Discoveries widget. We also looked at the horizontal scroll bar in the Data Catalog as a reference. Then we looked at the remaining images & content on the VEDA landing page. They liked all of these as options and now want to show something similar on the GHG landing page. They anticipate if we use the same components and update the content, that this will be close enough to what HQ is asking for that we can a) use these widgets for the beta release and b) spark some ideas from them on the actual content.

So, Deborah will be providing some content (words & images) to use in this first test. Then, the ask is for us to replicate what we have in VEDA on the GHG dashboard with this updated content. 

Note: for things like the Featured Discoveries section that relies on content elsewhere in the dashboard, Deborah said she is okay if we just include an image in place of the actual widget. Again, we're just looking for something quick as a catalyst for HQ to provide more content & direction.

Timing: IMPACT has a meeting with HQ the morning of Friday, August 11. They would like to have a demo link for this meeting. It does not have to be merged, netlify preview is okay.

**Comment by faustoperez** [comment]
Here's a mockup of the designs shared yesterday on the Slack Impact channel.

We decided to use just text for the GHG logo as a placeholder and feature the four agency logos more prominently.  

![US GHG Center - Landing page](https://github.com/NASA-IMPACT/veda-config-ghg/assets/12268629/f269d40f-85b9-4cdd-b644-d7fb522f16b5)

**Comment by Jeanne-le-Roux** [comment]
I showed the design mockup to HQ this morning. They thought it looked great and we are good to move forward, but it was asked that we change the top image to the following one:
https://svs.gsfc.nasa.gov/5115 

The screen grab right at the beginning of the video is fine since it features the U.S. 

We may also be asked to update the icons for the 3 focus areas to different images. I will keep you posted on any additional updates.

**Comment by Jeanne-le-Roux** [comment]
Another comment - the 3 icons for the focus areas will need to be larger in size. I'll get us a copy of the approved text so we can add that to the home page as well.

**Comment by faustoperez** [comment]
Thanks for the feedback @Jeanne-le-Roux, much appreciated, we'll send an updated version of the designs as soon as they're ready 👍

**Comment by Jeanne-le-Roux** [comment]
I've received some additional feedback. I'll try my best to summarize here but am also happy to tag up with someone to discuss:

- Request to increase the size of the 4 agency logos at the top (I didn't get specifics, just that they need to be "a bit bigger")

- For the 3 focus areas, as I already mentioned the icon/image sizes will need to be larger. These are also the images that have been selected for each:
Anthropogenic GHG Emissions
(Source: https://research.noaa.gov/tag/carbon-cycle/)
![img-AirPollution-1024x576](https://github.com/NASA-IMPACT/veda-config-ghg/assets/17416300/1619cf0c-6c90-494d-9a01-b2714a022c7e)

Natural GHG Sources and Sinks (source: https://www.fisheries.noaa.gov/feature-story/recommendations-reducing-wetland-loss-coastal-watersheds-united-states)
![hcarter6-marsh-sunny-aspect](https://github.com/NASA-IMPACT/veda-config-ghg/assets/17416300/e19566f4-d8ba-4fbe-8141-14b71aad8700)

Large Methane Emission Events (source: https://www.nasa.gov/feature/jpl/methane-super-emitters-mapped-by-nasa-s-new-earth-space-mission)
![1-pia25592_emit_methane_permian](https://github.com/NASA-IMPACT/veda-config-ghg/assets/17416300/49eb3e15-9d9b-43ea-b76f-975a0021c890)

- When clicking on the right arrow > next to each focus area, rather than have it link directly to a resource, have it open a small drop-down section where some info can be populated. Right now we want to highlight at least 2 resources for each focus area: 1) Link to the associated Data Story, 2) Link to sub-set of relevant datasets in the data catalog (can be done with 'Topic' filter). This can be done via some hyperlinked text. Let me know if this is feasible or if there are other suggestions on how to provide some additional info for each of the 3 focus areas on the home page when a user inspects it. If this isn't feasible, we may have to build 3 different sub-pages for each one that can be linked to.  

- This leaves a question of what to do with the static image just to the right of the 3 focus areas. Right now it is a plume image, but since this is the same image to be used for the Large Methane Emission Events focus area icon, it will need to be swapped out. Could we possibly make this more dynamic to show a different image in this spot if the user clicks on one of the 3 options? 

- There will be an overall GHG Center Data Story and we will have to feature this data story on the home page somehow. Let me know if you have recommendations on how to best highlight this Data Story (E.g. if the 3 focus areas already link to the data catalog, perhaps we could repurpose the blue "Browse the GHG Data Catalog" button. Open to other ideas though.)

- As a heads up we will probably be getting some feedback on the website footer. I think what you have for now is good (some text, agency logos, link to learn more & the various website components), but I wouldn't be surprised if we get asked to make some modifications. (see [Digital.gov](https://digital.gov/resources/required-web-content-and-links/?=checklist) and [USWDS Footer recommendations](https://designsystem.digital.gov/components/footer/)). Given the multi-agency nature of the project the guidance for just a single agency may not exactly apply. I imagine we'll have some leeway to update the footer throughout September as we get more official feedback. 

I'm still waiting on an update on the exact draft text to provide on the home page. As soon as I get access to it I will post a google doc with the draft text.

**Comment by faustoperez** [comment]
Hi @Jeanne-le-Roux and thanks a lot for the feedback. We have updated the designs:

- Substituted the hero globe with the one you provided.
- Increased the size of the agency logos.
- Updated the 3 focus areas with larger images / removed the icons and the hover behaviour so it's a more standard approach to reduce interaction complexity.
- We have also updated the designs to a card layout that allows highlighting resources in each focus area.

We have in mind the coming changes to the texts, we can tackle that directly in code.

We're also aware that there will be feedback on the footer and we'll need to have a featured Data Story block. We suggest working on a design solution for those after this release when we get more official feedback.

Thank you, and have a nice week!

<img width="1728" alt="US GHG Center - Landing page" src="https://github.com/NASA-IMPACT/veda-config-ghg/assets/12268629/fd2aa61c-5d29-495c-9114-7329cb825934">

**Comment by Jeanne-le-Roux** [comment]
Thank you, this is looking great! The card layout is nice. We can continue to iterate on the footer. Also just a minor request to center the text just below the image (where it says 'Start exploring scientist-curated...')

Here's [the doc](https://docs.google.com/document/d/1iKeai-Y_219GCoRiWy5q6mUmvBHt-HmniBUSWppEgVI) with draft text for the home page. 

Also just wanted to make sure the [hub tab](https://github.com/NASA-IMPACT/veda-architecture/issues/314) has been accounted for.

**Comment by j08lue** [comment]
Footer (draft) content: https://docs.google.com/document/d/1iKeai-Y_219GCoRiWy5q6mUmvBHt-HmniBUSWppEgVI/edit#heading=h.xyuh737klpu

**Comment by Jeanne-le-Roux** [comment]
@slesaad is working to add the draft Data Story content, hopefully we can push her updates soon so the data stories are accessible. 

I believe the updated About page/partnership text (also in the doc) includes a link to the GHG data story. I like the way the story is being linked on the home page in the preview link. We are dealing with 2 'about' content pieces (about page and GHG center story), however they include different information and it would probably be overwhelming to have both on the same page. We can think about how to better present this post R1 when we consider the overall site design. 

I'll have to get back to you on the contact info. As a heads up I will be off starting tomorrow (8/31 - 9/4) and @deborahUAH will be your main POC for anything home page related.

**Comment by Jeanne-le-Roux** [comment]
Also something I noticed - the very bottom of the footer currently hyperlinks to Earthdata.nasa.gov (where it says "By U.S. GHG Center on 2023 • v0.8.0"). This link needs to be removed.

**Comment by danielfdsilva** [comment]
@Jeanne-le-Roux link removed

**Comment by Jeanne-le-Roux** [comment]
We have Data Stories (/data insights) available now. Can these links be added to the home page in the cards for the 3 focus areas? Using text/linked stories below:
1.  [Read more about Anthropogenic GHG Emissions](https://ghg-demo.netlify.app/stories/anthropogenic-methane-emissions-distribution) 
2. [Read more about Natural GHG Sources and Sinks](https://ghg-demo.netlify.app/stories/how-ghg-center-tracks-carbon)
3. [Read more about how Large Methane Emission Events are identified](https://ghg-demo.netlify.app/stories/discovering-large-methane-emissions)

**Comment by danielfdsilva** [comment]
@Jeanne-le-Roux Links added. Clicking on a card will now take the user to the corresponding story.

**Comment by Jeanne-le-Roux** [comment]
@danielfdsilva Thank you. There's also been a request to center the Welcome text just below the blue banner image.

**Comment by Jeanne-le-Roux** [comment]
@danielfdsilva & that the 'Hub' link open to a new tab

**Comment by j08lue** [comment]
I created a new ticket - easier for tracking.
